bin/db-initialization.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+import os
+
+from buildpg import V, asyncpg, render
+
+DB_NAME = os.environ["DATABASE_TO_CREATE"]
+
+
+def db_initialization():
```

Comment:
Seems like this should be the async function, and just wrap the call in the main check instead of in this function.

Reply:
Also, this function is really long. It doesn't do a whole lot, but it doesn't look like it. I think breaking it down in to smaller, self-documenting functions would make the code more obvious.